src/components/MyReservation.js

```diff
@@ -20,7 +21,7 @@ const MyReservation = () => {
         Authorization: localStorage.getItem('token'),
       },
     };
-    fetch(`http://localhost:3000/reservations/${id}`, params)
+    fetch(`${url}reservations/${id}`, params)
```

Comment:
Changing the URL to a dynamic URL is a time-saving and DRY method of maintaining URLs for the future. Good call here!